src/components/Sections/Events/UnitItem.js

```diff
@@ -24,9 +25,9 @@ const UnitItem = ({ id, t }) => {
             <div className="additional_links">
               {
                 collections.map(c => (
-                  <Link key={c.id} to={canonicalLink(c)}>
+                  <span key={c.id}>
```

Comment:
@wissog  why you remove link to collection?

Reply:
it threw an html error because of link inside a link... and didn't contribute much in events - linking to the same collection but a different unit...
but you're right, i found a better way to fix this error